README.md

```diff
@@ -49,7 +49,7 @@ export const config: Config = {
   url: "https://www.builder.io/c/docs/developers",
   match: "https://www.builder.io/c/docs/**",
   selector: `.docs-builder-container`,
-  maxPagesToCrawl: 1000,
```

Comment:
I'd suggest maybe a value more like 50 for this

5 isn't really a crawl, so if someone without changing any options just plugs in a site and expects to product a GPT from it, 5 pages will do poorly, so I definitely want to guide people to the "pit of success" rather than a pit of failure

that said, 1000 is certainly high if anyone has concerns of too high of load, so maybe 50 or 100 is a decent compromise. noting also that the crawler only does so much concurrency too, so its not like all of these requests happen simultaneously